plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/ray_launcher.py

```diff
@@ -0,0 +1,211 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
```

Comment:
one thing to start paying attention to is plugin import time.
see [this](https://hydra.cc/docs/next/advanced/plugins#plugin-discovery).

You can the discovery time for the slowest plugins with `hydra.verbose=true`.
(we should mention this in the [debugging page](https://hydra.cc/docs/next/tutorials/basic/debugging) of the tutorial).